packages/widgets/src/menubar.ts

```diff
@@ -382,6 +391,74 @@ export class MenuBar extends Widget {
     }
   }
 
+  /**
+   * A message handler invoked on an `'resize'` message.
+   */
+  protected _evtResize(event: Event): void {
```

Comment:
No, there's not a resize event on the initial page load, so users will first have to resize their window. I'm not sure if there's a signal that exists when the page is loaded that I can connect this method to. If you know of it please let me know.

Reply:
@afshin @fcollonval is there a good signal for initial load?

This seems bad from an end user's perspective, doesn't it? For example, if I've set my browser to a higher zoom and then load the JupyterLab UI, as an end user I don't want to have to jiggle the window resize handle to get the menu bar to render properly.

That said, just to be clear I'm not asking us to block on this, because it's something that can be added later.